.github/workflows/code-health.yml

```diff
@@ -45,108 +45,40 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: 'go.mod'
-      - run: go install gotest.tools/gotestsum@latest
+      - run: |
+          go install gotest.tools/gotestsum@latest
+          go install github.com/mattn/goveralls@latest
```

Comment:
should we ping to a specific version? This could prevent the action to fails because of some changes to the latest release